python/hail/methods/statgen.py

```diff
@@ -281,22 +281,22 @@ def linear_regression(y, x, covariates=(), root='linreg', block_size=16) -> Matr
 
     .. math::
 
-        \mathrm{height} = \\beta_0 + \\beta_1 \, \mathrm{genotype}
-                          + \\beta_2 \, \mathrm{age}
-                          + \\beta_3 \, \mathrm{is_female}
+        \mathrm{height} = \\beta_0 \, \mathrm{genotype}
```

Comment:
There still is an intercept in the example. In my lmm branch, I've moved to using beta_\star for the effect of interest, e.g.:
```
\\beta_\star \, \mathrm{genotype} + \\beta_0 + \\beta_1 \, \mathrm{age} + \\beta_1 \, \mathrm{is_female}
```
I think this is cleaner and clearer, could you do the same in linreg and logreg equation/table (lmmreg and skat don't need it).

You could also replace all the`\\` in linreg with `\` if you put make the docstring raw, or leave it.